www/braj08/homework_03/main.css

```diff
@@ -0,0 +1,132 @@
+* {
+    box-sizing: border-box;
+}
+
+body {
+    background-color: white;
+    font-family: sans-serif;
+}
+
+.text-02 {
+    color: red;
+}
+
+.text-03 {
+    color: rgb(0, 255, 0);
+}
+
+.text-04 {
+    color: #0000FF;
+    font-weight: bold;
+}
+
+.text-05 {
+    font-size: 18px;
+}
+
+.text-06 {
+    font-size: 24px;
+    font-style: italic;   
+}
+
+.text-07 {
+    color: white;
+    background-color: brown;
+}
+
+.paragraph-01 {
+    text-align: center;
+    line-height: 32px;   
+}
+
+.paragraph-02 {
+    text-align: justify;
+    text-transform: uppercase;
+}
+
+.box-01 {
+    width: 400px;
+    height: 300px;
+    border-style: solid;
+    border-color: purple;
+    border-width: 1px;
+    padding: 30px;    
+}
+
+.box-02 {
+    width: 500px;
+    height: 200px;
+    background-color: rgb(121, 3, 3);
+    margin: 100px;
+}
+
+.box-03 {
+    display: none;
+}
+
+.box-04 {
+    width: 100%;
+    height: 150px;
+    background-color: rgb(0, 95, 219);
+    padding: 50px;
+    border-style: solid;
+    border-width: 10px;
+    border-color: yellow;
+}
+
+.box-05 {
+    visibility: hidden;
+}
+
+.box-06 {
+    width: 500px;
+    height: 200px;
+    background-color: rgba(0, 192, 58, 0.5);
+}
+
+.box-07 {
+    width: 400px;
+    height: 300px;
+    background-color: lightblue;
+    position: relative;
+}
+
+.box-child-01 {
+    width: 50px;
+    height: 50px;
+    position: absolute;
+    top: 50px;
+    right: 10px;
+    background-color: pink;
+    font-size: 11px;
+}
+
+.button-01 {
+    width: 300px;
+    height: 100px;   
+}
+
+.button-01:active {
+    color: blue;
+}
+
+.button-02 {
+    width: 200px;
+    height: 50px;   
+}
+
+.button-02:hover {
+    background-color: brown;
+}
+
+.box-08 {
+    width: 200px;
+    height: 500px;   
+    background-color: darkcyan;
+    margin-top: 200px;
+}
+
+.box-08:hover {
+    cursor: pointer;
```

Comment:
vlastnost **cursor: hover** nemusi byt v :**hover** pseudo tride, protoze jiz sama implikuje to ze element musi byt **hovered** aby to melo efekt